### PR TITLE
Version bump of mendeleydesktop

### PR DESCRIPTION
**Pull request by marbre**
Mendeleydesktop 1.13 was released, this is a version bumped ebuild.

Further, I updated the license file. As these [terms of use](http://www.mendeley.com/terms/) don't contain the word EULA anymore, I rename the license file to Mendeley-terms.
Actually I am not sure if this is the correct license. The tarball itself contains a *LICENSE* file in *work/mendeleydesktop-1.13/* and an identical *License.txt* file  in *work/mendeleydesktop-1.13/share/doc/mendeleydesktop/*, which does not get installed. These are quite similar to the terms, but **not identical**.

### CODE REVIEW

File: sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+QA_PREBUILT="/opt/mendeleydesktop/.*"
+
+pkg_nofetch() {
+	elog "Please download ${A} from:"
+	elog "http://www.mendeley.com/download-mendeley-desktop/"
+	elog "and move it to ${DISTDIR}"
+}
+
+src_unpack() {
```
Comment (junghans) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: I guess `src_unpack` from `inherit vcs-snapshot` would do the same.

Comment (marbre) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: *inherit vcs-snapshot* and afterwards calling *src_unpack* **or** calling *vcs-snapshot_src_unpack* explicitly in *src_unpack* results both in *work/mendeleydesktop-1.13-linux-x86_64*. That's not what I want.

Comment (junghans) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: I was thinking to remove `src_unpack` completely, which results in `vcs-snapshot_src_unpack` being called. But what I forgot was, that `vcs-snapshot.eclass` will unpack to the basename of `${A}`, so you would need to use the `->` operator, however that won't work as the tarball for x86 and amd64 need to be stored under different names.

So leave it like it is!

File: sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	if use amd64 || use amd64-linux ; then
+		mv -f "${MY_P_AMD64}" "${P}" || die
+	else
+		mv -f "${MY_P_X86}" "${P}" || die
+	fi
+}
+
+src_prepare() {
+	# remove bundled Qt libraries
+	rm -rf lib/mendeleydesktop/plugins \
```
Comment (junghans) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: `rm -f` will never fail, so either drop the `die` or remove the `-f`.

File: sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+LICENSE="Mendeley-terms"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RESTRICT="fetch"
+
+DEPEND=""
+RDEPEND="
+	>=dev-qt/qtcore-4.6:4
```
Comment (junghans) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: `dev-qt/qtcore-4.6` is not even in gx86 anymore, just replace it by `dev-qt/qtcore:4`.

File: sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild (hunk 2 above)
Comment (marbre) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: You're right. Removed the force option.

File: sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild (hunk 3 above)
Comment (marbre) [inline sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild]: Fixed that, also for the other qt packages.

**Pull request merged**